convert/gitlab/yaml/job_test.go

```diff
@@ -15,12 +15,76 @@
 package yaml
 
 import (
+	"io/ioutil"
+	"path/filepath"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 	"gopkg.in/yaml.v3"
 )
 
+func TestJobYaml(t *testing.T) {
```

Comment:
in this case we are testing parsing Pipelines, so we should move this to `pipeline_test.go`.  this will ensure consistency with the other yaml packages.

Reply:
done.